### ISSUE TITLE
MeshCentral Random Hosts / Machines Appearing

### ISSUE THREAD

**Issue by compumatter** [Ylianst/MeshCentral#5101]
**Is your feature request related to a problem? Please describe.**
I am finding random host machines appearing in my MeshCentral dashboard.  Since I hope to provide MeshCentral as a solution for my clients servers, this will scare the heck out of them.   It seems that AV companies are auto-testing these agent files and so they are only online for a brief period.  However the machine shows up permanently until deleted.

**Describe the solution you'd like**
Some kind of an auto-delete feature if an agent has not had more than X seconds of total activity, they should be auto removed.  This would seem like an important configuration to insure the dashboard does not get flooded with false machines over time.

**Describe alternatives you've considered**
I have no alternative idea on this.

**Additional context**
I have created a video to make my particular problem more clear

https://www.youtube.com/watch?v=7MVUG50Y6Ak

Thank you.

Jay
CompuMatter

**Comment by si458** [comment]
This feature already exists

Create a group, click the group name

Select edit next to features

Remove device on disconnect (does as it says)

Automatically remove inactive devices (again as it says, but sadly its days and not minues/hours)

**Comment by RichardBurt65** [comment]
The random hosts are coming from antivirus vendors. They spin up a VM, start up the mesh package and then scan it. 

You can safely delete them.

**Comment by davwoo** [comment]
Just to add some further context to this, I agree with the AV testing theory.

I added a Windows agent to a machine a few days ago and, when I installed the agent on the target machine the Avast AV flagged the install file then finally accepted it and allowed it to run.

The following day a "John PC" appeared in the same group as the machine I'd added. When I did a whois on the PC's IP address, is resolved to Avast's domain.
